.travis.yml

```diff
@@ -123,20 +123,24 @@ script:
        export PATH="/usr/local/opt/ccache/libexec:$PATH";
     fi
 
-  # Test Debug or RelWithDebInfo compilation
-  - mkdir bin; cd bin
+  - mkdir bin; cd bin;
   - $CC -v
   - $CXX -v
   - ccache --version
-  - cmake ..
-    -DCMAKE_BUILD_TYPE=$BUILD
-    -DMYSQL_MAINTAINER_MODE=$MAINTAINER_MODE
-    -DBUILD_CONFIG=mysql_release
-    -DFEATURE_SET=community
-    -DENABLE_DTRACE=OFF
-    -DWITH_SSL=$LIBTYPE
-    -DWITH_ZLIB=$LIBTYPE
-    $CMAKE_OPT
+
+  - echo -- Perform Debug or RelWithDebInfo compilation;
+    if [[ "$INVERTED" == "ON" ]]; then
+       CMAKE_OPT="-DWITH_SSL=system -DWITH_ZLIB=bundled $CMAKE_OPT";
```

Comment:
I would also add `-DWITH_READLINE=OFF` here which would mean that both `WITH_READLINE` and `WITH_LIBEDIT` would be `OFF` (meaning that no bundled code would be used) and one of the system libraries would be used instead.